lms/djangoapps/discussion/django_comment_client/tests/group_id.py

```diff
@@ -32,10 +32,10 @@ def _assert_comments_service_called_without_group_id(self, mock_request):
 
     def _assert_html_response_contains_group_info(self, response):
         group_info = {"group_id": None, "group_name": None}
-        match = re.search(r'"group_id": (\d*),', response.content)
+        match = re.search(r'"group_id": (\d*),', response.content.decode('utf-8'))
         if match and match.group(1) != '':
             group_info["group_id"] = int(match.group(1))
-        match = re.search(r'"group_name": "(\w*)",', response.content)
```

Comment:
This comma is supposed to be inside the regular expression.  Was it removed on purpose?

Reply:
yes. html is different there is no comma in py3